lib/vault/client.rb

```diff
@@ -325,5 +336,36 @@ def error(response)
 
       raise HTTPError.new(address, response.code, [response.body])
     end
+
+    # Execute the given block with retries and exponential backoff.
+    def with_retries(rescued = [Exception], &block)
```

Comment:
I'd just pull the arg default here since no callers use it. Seems ripe for our far-future selves to accidentally fall back to this behavior when we didn't mean to. :grinning: